.github/workflows/build.yaml

```diff
@@ -67,6 +67,7 @@ jobs:
       - name: install dependencies
         run: |
           ./install-build-deps.sh
+          sudo apt-get install -y linux-modules-extra-$(uname -r)
```

Comment:
Please don't add more stuff to build.yaml; instead fix the scripts so that we can recreate this outside of github.

Reply:
Fixed.